exrsplit/__main__.py

```diff
@@ -13,6 +13,8 @@ def _parse_args():
     parser.add_argument('-s', '--split-channels', action='store_true', help='Create a file for each channel instead ' +
                         'of per layer. Data channels (eg. depth, shadows or mask) are saved as grayscale ' +
                         'representations of their data.')
+    parser.add_argument('--layer', action='append', help='Layer or Layer.channel to split OpenEXR in')
```

Comment:
Can you reword to something like `Split only selected layers`?

Reply:
Sure!